### ISSUE TITLE
Pinstate monitor missing PINs

### ISSUE THREAD

**Issue by ingoiot** [STEMpedia/DabbleESP32#3]
Hi,

the Pinstate monitor only shows 2 analog PINs with wrong data,
but 99 digital i/o PINs.

what went wrong?

![unnamed](https://user-images.githubusercontent.com/52928035/84943258-657dd980-b0e4-11ea-8688-7c9101c9698e.jpg)
![unn1amed](https://user-images.githubusercontent.com/52928035/84943269-67479d00-b0e4-11ea-9058-95444ab5680f.jpg)

**Comment by mimansamaheshwari** [comment]
Yes, there is some issue here, working on it, will let you know as soon as possible.
